app/_data/docs_top_nav.yml

```diff
@@ -13,6 +13,9 @@
 - name: Kong Ingress Controller
   desc: Works inside a Kubernetes cluster and configures Kong to proxy traffic
   url: /kubernetes-ingress-controller/latest/
+- name: Kong Gateway Operator
```

Comment:
This pushes us on to three lines. @acgoldsmith you mentioned that you were going to drop one entry when KGO came along. Can you remember which?